debugger/src/cell_renderers/cellrendererbreakicon.h

```diff
@@ -40,14 +40,14 @@ struct _CellRendererBreakIcon
 {
   GtkCellRenderer parent;
 
-  guint GSEAL (enabled);
-  const gchar* GSEAL(condition);
-  guint GSEAL (hitscount);
+  guint enabled;
+  const gchar* condition;
+  guint hitscount;
 
-  GdkPixbuf *GSEAL (pixbuf_enabled);
-  GdkPixbuf *GSEAL (pixbuf_disabled);
-  GdkPixbuf *GSEAL (pixbuf_conditional);
-  GdkPixbuf *GSEAL (pixbuf_file);
+  GdkPixbuf *pixbuf_enabled;
+  GdkPixbuf *pixbuf_disabled;
+  GdkPixbuf *pixbuf_conditional;
+  GdkPixbuf *pixbuf_file;
```

Comment:
I guess this would ideally use a private member: the goal of `GSEAL()` during GTK3 transition preparation was to "hide" members to callers by mangling their name.  The "correct" solution is to use a private, opaque, structure.

Reply:
Whats the point of complicating things by making them "private"? This is a debugger plugin, not a library that needs to be protected from being used by millions.

Reply:
Yeah it probably doesn't matter much here indeed.  I just mentioned that because the code previously used `GSEAL()` that's all.

Reply:
Yes I didn't understand the reason to use GSEAL() in the first place in the GTK+2 code. It just broke the GTK+3 version.
As a more elegant solution to the private nature of these members, I moved the structure definitions out of the .h into the .c.